src/battle_script_commands.c

```diff
@@ -4093,6 +4093,9 @@ static void Cmd_getexp(void)
                 calculatedExp = gSpeciesInfo[gBattleMons[gBattlerFainted].species].expYield * gBattleMons[gBattlerFainted].level / 7;
             #endif
 
+            if (B_TRAINER_EXP_MULTIPLIER <= GEN_7 && gBattleTypeFlags & BATTLE_TYPE_TRAINER)
+                calculatedExp = (calculatedExp * 150) / 100;
```

Comment:
Why did you move this out of `ApplyExperienceMultipliers`?

Reply:
If you look at the formula that includes this (a here), it is done before the exp share multiplier. Moving it here is more accurate.
![image](https://github.com/rh-hideout/pokeemerald-expansion/assets/23617175/4b9dff80-911d-4622-b8ba-15821ea60a68)